src/User/Filter/TwoFactorAuthenticationEnforceFilter.php

```diff
@@ -38,8 +38,10 @@ public function beforeAction($action)
         }
 
         $permissions = $module->twoFactorAuthenticationForcedPermissions;
+
+        $user = Yii::$app->user->identity;
         $itemsByUser = array_keys($this->getAuthManager()->getItemsByUser(Yii::$app->user->identity->id));
```

Comment:
This could be simplified to 

```php
array_keys($this->getAuthManager()->getItemsByUser($user>id));
```
(cosmetic, feel free to igiore)

Reply:
Made the requested change